corehq/apps/hqmedia/templates/hqmedia/references.html

```diff
@@ -70,12 +70,19 @@
                     {% blocktrans %}
                         You can manage multimedia by uploading files <strong>individually</strong> below.
                     {% endblocktrans %}
+                    <br><br>
                     {% blocktrans %}
```

Comment:
`blocktrans` looks unnecessary here, just `trans` should be good enough?

Reply:
Yeah, `trans` would work. The translations on this page aren't great in general, the way that this paragraph is broken into multiple pieces that may not go back together properly. Given that it's ICDS only, I'm not that concerned, but feel free to push back on me if that sounds too lazy.